client/src/components/Profiles.js

```diff
@@ -1,43 +1,91 @@
 import React, { useState, useEffect } from "react";
-import { Container, Message, Search, Table, Button } from "semantic-ui-react";
+import {
+  Message,
+  Search,
+  Table,
+  Button,
+  Grid,
+  Dropdown,
+  Input,
+  Container
+} from "semantic-ui-react";
 import { Link } from "react-router-dom";
-import { getProfiles } from "../api/profiles";
+import { getProfiles, getNationalities } from "../api/profiles";
 
 const Profiles = () => {
   const [profiles, setProfiles] = useState([]);
+  const [nationality, setNationality] = useState([]);
```

Comment:
```suggestion
  const [nationalities, setNationalities] = useState([]);
```
I think this should be plural since it is a list of nationalities.